design/MonitoringModeAPI.md

```diff
@@ -1997,3 +2015,166 @@ Returns the recommendation at a particular timestamp if it exists
+                {
+                  "type": "info",
+                  "message": "Duration Based Recommendations Available"
+                }
+              ],
+              "data": {
+                "2023-01-02T00:15:00.000Z": {
+                  "duration_based": {
+                    "short_term": {
+                      "monitoring_start_time": "2023-01-01T00:15:00.000Z",
+                      "monitoring_end_time": "2023-01-02T00:15:00.000Z",
+                      "duration_in_hours": 24.0,
+                      "pods_count": 7,
+                      "confidence_level": 0.0,
+                      "config": {
+                        "limits": {
+                          "cpu": {
+                            "amount": 0.9299999999999999,
+                            "format": "cores"
+                          },
+                          "memory": {
+                            "amount": 238.2,
+                            "format": "MiB"
+                          }
+                        },
+                        "requests": {
+                          "cpu": {
+                            "amount": 0.9299999999999999,
+                            "format": "cores"
+                          },
+                          "memory": {
+                            "amount": 238.2,
+                            "format": "MiB"
+                          }
+                        }
+                      },
+                      "variation": {
+                        "limits": {
+                          "cpu": {
+                            "amount": 0.42999999999999994,
+                            "format": "cores"
+                          },
+                          "memory": {
+                            "amount": 138.2,
+                            "format": "MiB"
+                          }
+                        },
+                        "requests": {
+                          "cpu": {
+                            "amount": -0.17000000000000015,
+                            "format": "cores"
+                          },
+                          "memory": {
+                            "amount": 187.98999999999998,
+                            "format": "MiB"
+                          }
+                        }
+                      },
+                      "notifications": []
+                    },
+                    "medium_term": {
+                      "pods_count": 0,
+                      "confidence_level": 0.0,
+                      "notifications": [
+                        {
+                          "type": "info",
+                          "message": "There is not enough data available to generate a recommendation."
+                        }
+                      ]
+                    },
+                    "long_term": {
+                      "pods_count": 0,
+                      "confidence_level": 0.0,
+                      "notifications": [
```

Comment:
Notifications is in older format, please update with the latest json

Reply:
done